requirements.md

```diff
@@ -72,13 +70,27 @@ For external access to the Anbox Cloud Appliance, a couple of network ports must
 
 How to allow incoming traffic on the listed ports is different depending on the cloud used. Please consult the documentation of the cloud for further information on how to change the firewall.
 
-## Juju based deployments
+<a name="juju-based"></a>
+## Juju-based deployments
 
 Anbox Cloud deployments are managed by Juju. They can be created on all the [supported clouds](https://juju.is/docs/clouds) as well as manually provided machines as long as they follow the required minimums.
 
-### Juju setup
+<a name="juju-version"></a>
+### Juju version
+
+Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. The latest supported version is **2.9**.
+
+[note type="information" status="Note"]If you are using Anbox Cloud 1.11 or earlier, the required Juju version is **2.8**.[/note]
+
+You can install Juju 2.9 with the following command:
+
+    snap install --channel=2.9/stable juju
```

Comment:
```suggestion
    snap install --classic --channel=2.9/stable juju
```

Reply:
Thanks - is this only for the install or also for refresh?

Reply:
It's only for installation. Refreshing doesn't need user acknowledgement for the classic mode, as it was already granted on installation